desktop/src/roles/role-list/role-list.component.scss

```
@@ -0,0 +1,308 @@
+@use "sass:map";
+@use "../../variables.scss" as variables;
+
+// Local aliases mapping the design tokens to the app palette.
+$fg-muted: map.get(variables.$accent-palette, 500);
+$fg-strong: map.get(variables.$accent-palette, 900);
+$border: rgba(0, 0, 0, 0.06);
+// Violet "Group" scope accent — intentionally outside the brand palette (the
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)